tests/baselines/reference/smartSelection_JSDocTags5.baseline

```diff
@@ -0,0 +1,43 @@
+/**
+ * @callback Foo
+ * @param {string} data
+ * @param {/**/number} [index]
```

Comment:
@andrewbranch What do we need to select in this case?

1. `number`
2. `@param {/**/number} [index]`
3. ?